terraform/modules/ecs/ecs.tf

```diff
@@ -417,6 +468,17 @@ resource "aws_ecs_task_definition" "reindex_vacancies_task" {
   task_role_arn            = "${aws_iam_role.ecs_execution_role.arn}"
 }
 
+resource "aws_ecs_task_definition" "performance_platform_submit_all_task" {
+  family                   = "${var.ecs_service_task_name}_performance_platform_submit_all_task"
+  container_definitions    = "${data.template_file.performance_platform_submit_all_container_definition.rendered}"
+  requires_compatibilities = ["EC2"]
+  network_mode             = "bridge"
+  cpu                      = "256"
```

Comment:
I know very little about Terraform but this seems like a high number, is it correct?

Reply:
I just copy pasted one of the existing one-off task definitions, the are all identiacal and use cpu 256 and memory 512.
@Stretch96 can you advise on this?

Reply:
256 CPU and 512 Memory is what we provision a container running Puma based only on a sensible default and our initial provision testing. Our instances running on non-prod environments are running t2.small instances which get 2Gb of memory and 2Ghz of CPU.

For this we want to choose a number that gives enough resources for the rake task to succeed (some can be quite hungry) but not too much that it might starve the other Puma containers of resources and fail to start. The EC2 servers will now scale if this load is detected but it won't have enough time to respond to a rake task so it will likely fail with a provisioning error from ECS if you specify this as for example: 2Gb.

Reply:
There's no reason this particular task couldn't be lower as 128 CPU however we _have_ had problems with the schools:import task running out of CPU. Once we found that 256 worked it seems easy to keep it as out default as there are no notable down sides.